### ISSUE TITLE
Unable to use asciidoctorj

### ISSUE THREAD

**Issue by aruizca** [jbake-org/jbake-gradle-plugin#10]
I replace the asciidoctor-java integration dependency with the asciidoctorj one and it does not work.

My ```build.gradle```:

```groovy
buildscript {
    repositories {
        mavenLocal()
        mavenCentral()
        jcenter()
    }

    dependencies {
        classpath 'me.champeau.gradle:jbake-gradle-plugin:0.2'
        classpath 'org.pegdown:pegdown:1.4.2'
//        classpath 'org.asciidoctor:asciidoctor-java-integration:0.1.4'
        classpath 'org.asciidoctor:asciidoctorj:1.5.2'
    }
}

apply plugin: 'me.champeau.jbake'

jbake {
    input  = file 'jbake'
    clearCache = true
    configuration['render.tag'] = 'true'
}
```

**Comment by jonbullock** [comment]
Try using Asciidoctorj v1.5.1 instead, I've not tried JBake with v1.5.2 yet.

**Comment by aruizca** [comment]
I get an error too. I am running on OSX 10.10
```
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':jbake'.
> org.asciidoctor.Asciidoctor.readDocumentHeader(Ljava/io/File;)Lorg/asciidoctor/DocumentHeader;
```

**Comment by jonbullock** [comment]
So you get an error when using v1.5.1 of AsciidoctorJ as a dependency as well?

**Comment by aruizca** [comment]
Yes. It might be an issue of Asciidoctorj with OSX

**Comment by jonbullock** [comment]
Add the following under the dependencies:

```
classpath 'org.jbake:jbake-core:2.3.2'
```
The current release of the plugin is defining a dependcy of v2.3.0 of JBake. By adding the above line you can then use v1.5.1 of AsciidoctorJ.

**Comment by aruizca** [comment]
It worked with 1.5.1 and 1.5.2!! Thanks heaps Jonathan!! Here is my build.gradle:
```groovy
buildscript {
    repositories {
        mavenLocal()
        mavenCentral()
        jcenter()
    }

    dependencies {
        classpath 'me.champeau.gradle:jbake-gradle-plugin:0.2'
        classpath 'org.jbake:jbake-core:2.3.2'
        classpath 'org.pegdown:pegdown:1.4.2'
//        classpath 'org.asciidoctor:asciidoctor-java-integration:0.1.4'
        classpath 'org.asciidoctor:asciidoctorj:1.5.2'
    }
}

apply plugin: 'me.champeau.jbake'

jbake {
    input  = file 'jbake'
    clearCache = true
    configuration['render.tag'] = 'true'
}
```

**Comment by jonbullock** [comment]
Glad to hear it works with v1.5.2 too.

**Comment by aalmiray** [comment]
I believe this issue can be closed as #7 allows the plugin to handle `asciidoctorj` for newer versions of `jbake` and `asciidoctorJavaIntegration` for older versions,
